strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/main/java/org/carlspring/strongbox/repository/NpmRepositoryManagementStrategy.java

```diff
@@ -29,24 +31,31 @@ protected void createRepositoryInternal(Storage storage,
     {
         String storageId = storage.getId();
         String repositoryId = repository.getId();
+        NpmRepositoryConfiguration repositoryConfiguration = (NpmRepositoryConfiguration) repository.getRepositoryConfiguration();
```

Comment:
If we use here `NpmRepositoryConfiguration`, the method argument can be `NpmRepositoryConfiguration` type instead of `Repository`. With that we can omit casting.